Documentation/compatibility/GetECDsaPublicKey-correctly-handles-brainpool-curves.md

```diff
@@ -0,0 +1,25 @@
+## ECDsaCertificateExtensions.GetECDsaPublicKey correctly handles Brainpool curves
+
+### Scope
+Minor
+
+### Version Introduced
+4.8
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+Starting with the .NET Framework 4.8, the `System.Security.Cryptography.X509Certificates.ECDsaCertificateExtensions.GetECDsaPublicKey(this System.Security.Cryptography.X509Certificates.X509Certificate2 certificate)` will now correctly handle Brainpool elliptic curves.
```

Comment:
Is it useful to add a one-sentence description of the previous behavior? ("In .NET Framework 4.7.2 and earlier versions...")

Reply:
previously it just didn't work with this API :smile: